docs/concepts/json_schema.md

```diff
@@ -1,22 +1,44 @@
 ??? api "API Documentation"
     [`pydantic.json_schema`][pydantic.json_schema]<br>
 
-Pydantic allows automatic creation of JSON schemas from models.
-
-Using Pydantic, there are several ways to generate JSON schemas or JSON representations from fields or models:
-
-* [`BaseModel.model_json_schema`][pydantic.main.BaseModel.model_json_schema] returns a jsonable dict of the schema.
-* [`BaseModel.model_dump_json`][pydantic.main.BaseModel.model_dump_json] returns a JSON string representation of the
-    dict of the schema.
-* [`TypeAdapter.dump_json`][pydantic.type_adapter.TypeAdapter.dump_json] serializes an instance of the adapted type to
-    JSON.
-* [`TypeAdapter.json_schema`][pydantic.type_adapter.TypeAdapter.json_schema] generates a JSON schema for the adapted type.
-
+Pydantic allows automatic creation and customization of JSON schemas from models.
 The generated JSON schemas are compliant with the following specifications:
 
 * [JSON Schema Draft 2020-12](https://json-schema.org/draft/2020-12/release-notes.html)
 * [OpenAPI extensions](https://github.com/OAI/OpenAPI-Specification).
 
+## Generating JSON Schema
+
+Use the following functions to generate JSON schema:
+
+* [`BaseModel.model_json_schema`][pydantic.main.BaseModel.model_json_schema] - returns a jsonable dict of a model's schema
+* [`TypeAdapter.json_schema`][pydantic.type_adapter.TypeAdapter.json_schema] - returns a jsonable dict of an adapted type's schema
+
+!!! note
+    These methods are not to be confused with [`BaseModel.model_dump_json`][pydantic.main.BaseModel.model_dump_json]
+    and [`TypeAdapter.dump_json`][pydantic.type_adapter.TypeAdapter.dump_json], which serialize instances of the
+    model or adapted type, respectively. These methods return JSON strings.
+
+!!! note "on the "jsonable" nature of JSON schema"
```

Comment:
How does this look? Cause the first double quotes is closed on the beginning of "jsonable".

Reply:
Looks good, but good idea to check.